regolith/helpers/a_expensehelper.py

```diff
@@ -93,84 +89,148 @@ def construct_global_ctx(self):
-                         })
-    # Updates the grants, assumes equal percentage for each grant
+            expense_type = 'travel'
+        pdoc.update({'expense_type': expense_type})
+
         if rc.grants:
-            if isinstance(rc.grants, str):
-                rc.grants = [rc.grants]
-            percentages = [100 / len(rc.grants) for i in rc.grants]
+            percentages = [100/len(rc.grants) for i in rc.grants]
+
+
         else:
-            rc.grants = 'tbd'
             percentages = [100]
+        pdoc.update({'grant_percentages':percentages,
+                     'grants': rc.grants})
+        if expense_type == 'travel':
+            pdoc.update({'itemized_expenses': [
+                {
+                    'date': date,
+                    'purpose': 'registration',
+                    'unsegregated_expense': 0,
+                    'segregated_expense': 0
+                },
+                {
+                    'date': date,
+                    'purpose': 'home to airport',
+                    'unsegregated_expense': 0,
+                    'segregated_expense': 0
+                },
+                {
+                    'date': date,
+                    'purpose': 'flights',
+                    'unsegregated_expense': 0,
+                    'segregated_expense': 0
+
+
+                },
+                {
+                    'date': date,
+                    'purpose': 'airport to hotel',
+                    'unsegregated_expense': 0,
+                    'segregated_expense': 0
+                },
+                {
+                    'date': date,
+                    'purpose': 'hotel',
+                    'unsegregated_expense': 0,
+                    'segregated_expense': 0
+                },
+                {
+                    'date': date,
+                    'purpose': 'hotel to airport',
+                    'unsegregated_expense': 0,
+                    'segregated_expense': 0
+                },
+                {
+                    'date': date,
+                    'purpose': 'airport to home',
+                    'unsegregated_expense': 0,
+                    'segregated_expense': 0
+                },
+                {
+                    'date': date,
+                    'purpose': 'meals',
+                    'unsegregated_expense': 0,
+                    'segregated_expense': 0
+                }
+                ]
+            })
+        else:
+            pdoc.update({'itemized_expenses': [
+                {
+                    'date': date,
+                    'purpose': rc.purpose,
+                    'unsegregated_expense': 0,
+                    'segregated_expense': 0
+                }]
+            })
+        pdoc.update({'notes': rc.notes,
+                     'overall_purpose': rc.purpose,
+                     'payee': rc.payee,
+                     })
+        if rc.status == 'submitted':
+            submission_day = begin_date.day
```

Comment:
we are moving to dates rather than year month day so maybe if we are updating we should update with ``submission_date``.  It may need a schema update.

Reply:
done